docs/openstack-servers.md

```diff
@@ -0,0 +1,150 @@
+# Openstack Servers
+
+To read more about Openstack Servers using the [upstream docs](https://docs.openstack.org/python-openstackclient/latest/cli/command-objects/server.html).
+
+#### List and view servers
+
+``` shell
+$ openstack server list
+    [--quote {all,minimal,none,nonnumeric}]
+    [--reservation-id <reservation-id>]
+    [--ip <ip-address-regex>]
+    [--ip6 <ip-address-regex>]
+    [--name <name-regex>]
+    [--instance-name <server-name>]
+    [--status <status>]
+    [--flavor <flavor>]
+    [--image <image>]
+    [--host <hostname>]
+    [--all-projects]
+    [--project <project>]
+    [--project-domain <project-domain>]
+    [--user <user>]
+    [--user-domain <user-domain>]
+    [--long]
+    [-n]
+    [--marker <server>]
+    [--limit <num-servers>]
+    [--deleted]
+    [--changes-since <changes-since>]
+```
+
+#### Create a new server
+
+``` shell
+$ openstack server create
+    (--image <image> | --volume <volume>)
+    --flavor <flavor>
+    [--security-group <security-group>]
+    [--key-name <key-name>]
+    [--property <key=value>]
+    [--file <dest-filename=source-filename>]
+    [--user-data <user-data>]
+    [--availability-zone <zone-name>]
+    [--block-device-mapping <dev-name=mapping>]
+    [--nic <net-id=net-uuid,v4-fixed-ip=ip-addr,v6-fixed-ip=ip-addr,port-id=port-uuid,auto,none>]
+    [--network <network>]
+    [--port <port>]
+    [--hint <key=value>]
+    [--config-drive <config-drive-volume>|True]
+    [--min <count>]
+    [--max <count>]
+    [--wait]
+    <server-name>
+```
+
+#### Delete a server
+
+``` shell
+$ openstack server delete [--wait] <server> [<server> ...]
+```
+
+# Launch a server from a snapshot
+
+#### Create a snapshot of the instance
+
+!!! note
+
+    If necessary, list the instances to view the instance name with the list server command above.
+
+1. Shut down the source VM before you take the snapshot to ensure that all data is flushed to disk. Use the openstack server stop command to shut down the instance:
+
+    ``` shell
+    $ openstack server stop myInstance
+    ```
```

Comment:
You need a space here. Otherwise it gets rendered under 1 and the numbering will be off.

Reply:
Added a space, it looks to be rendering correctly in my preview.